taxonomy/utils.py

```diff
@@ -347,6 +355,7 @@ def refresh_product_skills(products, should_commit_to_db, product_type):
         skipped_count,
         len(all_failures),
     )
+    return success_count, len(all_failures)
```

Comment:
this utility is being called at many places. Will this return statement have any impact on those places?

Reply:
@sameenfatima78 No, as none of the callers are using it return value.